bibliography.bib

```diff
@@ -251,148 +125,82 @@ @misc{world_nuclear_association_nuclear_2017
-	doi = {10.1016/j.egypro.2011.06.038},
-	abstract = {Our world is ever growing, there will be higher demands on electricity, and fossil fuels cannot satisfy this demand without further harming the environment. Likewise, renewable energy sources such as solar and winds are still in their infancy and, when used alone, are not realistic solutions to meet this demand. Westinghouse Electric Company is ready to address higher electricity demand with the only Generation III+reactor to receive Design Certification from the United States Nuclear Regulatory Commission, the AP1000{\texttrademark} pressurized water reactor (PWR). Westinghouse Electric Company once again sets a new industry standard with the AP1000{\texttrademark} reactor. The AP1000{\texttrademark} is a two-loopPressurized Water Reactor (PWR) with passive safety features and extensive plant simplifications that enhance its construction, operation, maintenance and safety. The paper discusses the unique design features of AP 1000.},
-	urldate = {2017-05-12},
-	journal = {Energy Procedia},
-	author = {Sutharshan, Balendra and Mutyala, Meena and Vijuk, Ronald P and Mishra, Alok},
-	month = jan,
-	year = {2011},
-	keywords = {Generation III+ reactor, core operating parameters, modular design, passive safety, probabilistic risk assessment, PWR},
-	pages = {293--302},
-	file = {ScienceDirect Full Text PDF:/Users/khuff/Zotero/storage/2D66N8K4/Sutharshan et al. - 2011 - The AP1000TM Reactor Passive Safety and Modular D.pdf:application/pdf;ScienceDirect Snapshot:/Users/khuff/Zotero/storage/E87IC8RM/S1876610211015475.html:text/html}
-}
-
-@techreport{cne2_reports_2015,
-	title = {Reports of the {CNE}2},
-	url = {https://www.cne2.fr/index.php/en/cne-2-2007-to-this-day},
-	abstract = {According to the provisions of the 2006 act, the long-term management of high and intermediatelevel
-waste involves two related components: the partitioning-transmutation of actinides found in the
-spent fuel of future nuclear reactors and the geological disposal of long-lived high and intermediatelevel
-waste (LLHLW \& LLILW) in accordance with the principle of reversibility. In addition, facilities
-in the front-end and back-end of the nuclear fuel cycle and the dismantling of decommissioned
-facilities produce long-lived low-level waste (LLLLW), very low-level waste (VLLW) and waste
-with augmented natural radioactivity. The LLLLW pose different management problems due to the
-very large quantities produced. Short-lived low and intermediate-level waste is stored at the Aube
-storage centre (centre de stockage de l{\textquoteright}Aube {\textendash} CSA)},
-	language = {English},
-	urldate = {2017-06-24},
-	institution = {Commission Nationae D'Evaluation},
-	author = {CNE2},
-	month = jun,
-	year = {2015},
-	pages = {120},
-	file = {CNE2 - Reports of the CNE2 - 2007 to this day:/Users/khuff/Zotero/storage/9WF2G3QN/cne-2-2007-to-this-day.html:text/html}
+@techreport{piet_implications_2010,
```

Comment:
The reason why there are items that I don't cite is because I have a lot of papers in `2017-bae-europe` zotero folder and I just export them. I include papers that I add to read and I never end up citing them in the paper. I will get rid of papers I don't cite in the bib.